infrastructure/application/package.json

```diff
@@ -5,17 +5,24 @@
   },
   "dependencies": {
     "@nodelib/fs.walk": "^1.2.8",
-    "@pulumi/aws": "^5.6.0",
+    "@pulumi/aws": "^5.42.0",
     "@pulumi/awsx": "^0.40.0",
     "@pulumi/cloudflare": "^4.7.0",
     "@pulumi/docker": "^3.2.0",
     "@pulumi/postgresql": "^3.4.0",
-    "@pulumi/pulumi": "^3.74.0",
+    "@pulumi/pulumi": "^3.75.0",
     "@pulumi/random": "^4.8.2",
     "@types/mime": "^2.0.3",
     "mime": "^3.0.0",
     "tldjs": "^2.3.1"
   },
+  "scripts": {
+    "setup-sandbox-env": "REACT_APP_GOOGLE_OAUTH_OVERRIDE=https://api.editor.planx.dev REACT_APP_API_URL=https://api.planx.in REACT_APP_HASURA_URL=https://hasura.planx.in/v1/graphql REACT_APP_SHAREDB_URL=wss://sharedb.planx.in",
```

Comment:
When running build commands, the local `.env` file is being picked up and used to populate these values.

These values are populated in as part of the current CI process for staging and production.